GradedModulePresentationsForCAP/gap/GradedModulePresentations.gi

```diff
@@ -926,6 +954,70 @@ InstallGlobalFunction( ADD_GRADED_IDENTITY,
     
 end );
 
+##
+InstallGlobalFunction( ADD_GRADED_LIFT_ALONG_MONOMORPHISM,
+
+  function( category )
+    
+    AddLiftAlongMonomorphism( category,
+
+      function( iota, tau )
+        local lift;
+
+        lift := LiftAlongMonomorphism( UnderlyingPresentationMorphism( iota ), UnderlyingPresentationMorphism( tau ) );
+        
+        if lift = fail then
+
+            return fail;
+
+        fi;
+
+        lift := GradedPresentationMorphism( Source( tau ), lift, Source( iota ) );
+
+        if not IsWellDefined( lift ) then
```

Comment:
In the case of lifts along monomorphisms, a computed non-graded lift has automatically to be graded. The same holds for colifts along epimorphisms.

Reply:
You are right, since the kernel iota is zero. This checks will be removed anyway when the PR 
https://github.com/homalg-project/homalg_project/pull/222
is merged to homalg.

Reply:
You are right, I've updated the PR.